pkg/investigations/mustgather/mustgather.go

```
@@ -192,6 +198,78 @@ func (c *Investigation) Name() string {
+		logging.Warnf("Firing alerts: failed to fetch - %v", err)
+		return
+	}
+
+	relevant := filterRelevantAlerts(alerts)
+	if len(relevant) == 0 {
+		return
+	}
+
+	var summary strings.Builder
+	summary.WriteString(fmt.Sprintf("Firing alerts at time of must-gather (%d):\n", len(relevant)))
+	for _, sev := range []string{"critical", "warning", "error"} {
```

golangci-lint (2.12.2)

[error] string `critical` has 5 occurrences, make it a constant

(goconst)